src/openpgp.js

```diff
@@ -158,7 +158,7 @@ export function decryptKey({ privateKey, passphrase }) {
 export function encrypt({ data, publicKeys, privateKeys, passwords, filename, armor=true }) {
   checkData(data); publicKeys = toArray(publicKeys); privateKeys = toArray(privateKeys); passwords = toArray(passwords);
 
-  if (!nativeAEAD() && asyncProxy) { // use web worker if web crypto apis are not supported
+  if (asyncProxy) { // use web worker when available
```

Comment:
let's enable the transferrable but i repeate my point that i think that resilience and not long running scripts is tha way, but i may be wrong and i'm up to your suggestions.